Pd 849 Syncthing Migration Followup (#2441)
truenas/documentation: content/Solutions/MigratingToTrueNAS.md

### SMB Migration

We recommend migrating via SMB sharing if you use computers with Windows OS (or if you prefer SMB).

> [Robocopy](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/robocopy) is ideal for users with Windows clients. SMB also allows you to drag & drop files to migrate them from your current NAS storage into TrueNAS. Using SMB will be slower than using a utility like Robocopy.

Reviewer: don't use the &, change to and or even drag-and-drop is okay. and don't use "will" keep it active voice and present tense

> [Robocopy](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/robocopy) is ideal for users with Windows clients. SMB also allows you to drag & drop files to migrate them from your current NAS storage into TrueNAS. Using SMB will be slower than using a utility like Robocopy.

Reviewer: don't use "will" keep it active voice present tense
```suggestion
[Robocopy](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/robocopy) is ideal for users with Windows clients. SMB also allows you to drag and drop files to migrate them from your current NAS storage into TrueNAS. Using SMB is slower than using a utility like Robocopy.
```
